src/background.js

```diff
@@ -507,12 +517,13 @@ async function blockFacebookSubResources (requestDetails) {
 
   const urlIsFacebook = isFacebookURL(requestDetails.url);
   const originUrlIsFacebook = isFacebookURL(requestDetails.originUrl);
+  const frameAncestorUrlIsFacebook = getFrameAncestorsURLAndCheckIfFacebook(requestDetails.frameAncestors);
 
   if (!urlIsFacebook) {
     return {};
   }
```

Comment:
Perf optimization could be to `return {}` ASAP for each individual check?

```suggestion
  const urlIsFacebook = isFacebookURL(requestDetails.url);
  // If this request isn't going to Facebook, let's return {} ASAP
  if (!urlIsFacebook) {
    return {};
  }
  const originUrlIsFacebook = isFacebookURL(requestDetails.originUrl);



```